build-openresty.sh

```diff
@@ -29,14 +29,28 @@ mkdir -p /tmp/build/usr/local/kong
 mkdir -p /tmp/build/
 mkdir -p /work
 
-if [ "$DEBUG" == 1 ]
+if [ -z "$KONG_NGINX_MODULE" ]
 then
-  KONG_NGX_BUILD_ARGS="--debug"
```

Comment:
This doesn't seem to have been the only usage of `$KONG_NGX_BUILD_ARGS`: https://github.com/Kong/kong-build-tools/blob/2021ffdbb67dc6957f974e49f6f58b116f226aab/build-openresty.sh#L77